src/Exporter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I'm assuming we are planning on declaring strict types for each of our files?  This seems reasonable to me at first glance, just wanted to get a temperature check.

Reply:
Yes, i think that gives us some advantages for non-typed language